discover/faq.md

```diff
@@ -47,7 +47,7 @@ Any type of data can be sold via Ocean Protocol. AI data, music data, image, vid
 
 <summary>How can we ensure the validity of datasets on the market?</summary>
 
-We are not actively developing a dedicated system for dataset validation. Instead, we have built tools and established multiple methods within our verification layer. However, the primary responsibility for data validation lies with the customer. To assess the data's quality, consider these questions: 
+We have built tools and established multiple methods within our verification layer. However, the primary responsibility for data validation lies with the customer. To assess the data's quality, consider these questions: 
```

Comment:
Well I don't think we have some concurrent methods for data verification but ways through which the user can validate or verify the dataset ( already written in the answer

Reply:
my point is I would remove this sentence `We have built tools and established multiple methods within our verification layer.`, since from my knowledge is false, but i may be wrong. The rest of the answer is good 👍

Reply:
hmm makes sense